pkg/controllers/nodeclaim/lifecycle/registration_test.go

```diff
@@ -204,7 +204,52 @@ var _ = Describe("Registration", func() {
 			Expect(node.Annotations).To(HaveKeyWithValue(k, v))
 		}
 	})
-	It("should sync the taints to the Node when the Node comes online", func() {
+	It("should not sync the taints to the Node when the Node comes online, with node label do not sync taints", func() {
+		nodeClaim := test.NodeClaim(v1.NodeClaim{
```

Comment:
Don't we need to the label added to the nodeclaim?

Reply:
I don't think that is required by any piece of code. We are looking at the node, not the nodeclaim to make the decision to sync or not to sync.

Reply:
Ahh your right, I still see that we don't add the label on the node to validate they taint operation. Doesn't the label need to be added the node?